cordial_gui/scripts/gui_controller.py

```diff
@@ -178,7 +178,8 @@ def cb(self, msg):
                 if s is not None:
                     s.unregister()
         else:
-            rospy.sleep(3)
+            seconds_to_sleep_for_tests = 3
```

Comment:
This is fine this time, but in general, try to avoid changes that don't go with the theme of the pull request.